NuGet.config

```diff
@@ -10,6 +10,14 @@
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-emsdk -->
     <add key="darc-pub-dotnet-emsdk-78f6f07" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-8/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-7/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-6/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-5/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-4" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-4/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-3" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-3/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-2" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-2/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-78f6f07-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-78f6f07d-1/nuget/v3/index.json" />
```

Comment:
@dotnet/area-infrastructure-libraries just curious what all these many feeds are being added for.